packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -1779,6 +1779,38 @@ void main() {
       lessThan(600 - windowPaddingBottom), // Device height is 600.
     );
   });
+
+  testWidgets('iconSize parameter tests', (WidgetTester tester) async {
+    final PopupMenuItemBuilder<PopupMenuItem<String>> itemBuilder = (BuildContext _) {
```

Comment:
I think this could be coded a little bit more simply. Something like:

```dart
Widget buildFrame({ double iconSize }) {
  return MaterialApp(
    home: Scaffold(
      body: Center(
        child: PopupMenuButton<String>(
          iconSize: iconSize,
          return <PopupMenuEntry<String>>[
            const PopupMenuItem<String>(
              value: 'value',
              child: Text('child'),
            ),
          ],
        ),
      ),
    ),
  );
}

await buildFrame();
expect(tester.widget<IconButton>(find.byType(IconButton)).iconSize, 24);

await buildFrame(iconSize: 50);
expect(tester.widget<IconButton>(find.byType(IconButton)).iconSize, 50);